reach

```diff
@@ -1,5 +1,14 @@
 #!/bin/sh
 
+if [ ! "$REACH_ALLOW_SU" = '1' ] && [ "$(id -u)" -eq 0 ]; then
+  echo "Reach isn't intended to be run with superuser privileges."
+  echo
+  echo "Please try again while logged into a normal, non-root account, and without using \
+\`sudo\`, \`su\`, or \`doas\` to grant elevated permissions."
+  echo
```

Comment:
```suggestion
  echo
  echo "If you think you need to do this, then maybe you\
really need to add yourself to the Docker group."
  echo
```

I believe this is [Jay's suggestion](https://github.com/reach-sh/reach-lang/pull/673#pullrequestreview-906688362).

Reply:
It's a little more complicated than that, unfortunately. I'm about to push an update if you're curious.